addon/-legacy-private/system/store.js

```diff
@@ -1871,7 +1871,7 @@ Store = Service.extend({
       snapshot: snapshot,
       resolver: resolver,
     });
-    emberRun.once(this, this.flushPendingSave);
+    emberRun.scheduleOnce('actions', this, this.flushPendingSave);
```

Comment:
What's this about

Reply:
Now that we have setup waiters for our backburner instance, we no longer need the autorun assertion. This allows folks (including ourselves) to use `async await` without wrapping everything in run loops.